tests/test_bug_features.py

```diff
@@ -72,7 +72,7 @@ def test_has_github_url():
 
 
 def test_whiteboard():
-    read('whiteboard.json', whiteboard, [['memshrink'], []])
+    read('whiteboard.json', whiteboard, [['memshrink', 'platform-rel-facebook'], []])
```

Comment:
Please add some additional tests, for example from the two bugs that I mentioned.